apps/mobile/src/components/ModifyOffer/atoms/offerFormStateAtoms.ts

```diff
@@ -411,10 +407,19 @@ export const offerFormMolecule = molecule(() => {
     optic.prop('offerDescription')
   )
 
-  const singlePriceStateAtom = getAtomWithNullableValueHandling(
-    nullableSinglePriceStateAtom,
-    offerFormAtom,
-    'singlePriceState'
+  const _singlePriceActiveAtom = atom<boolean>(true)
```

Comment:
Let's remove `_` in the naming, since this atom is used outside of toggleSinglePriceActiveAtom.